.travis.yml

```diff
@@ -15,10 +15,3 @@ before_install:
 script:
 - bundle exec bin/rails app:test:providers:amazon:setup app:test:providers:amazon
 after_script: bundle exec codeclimate-test-reporter
-notifications:
-  webhooks:
-    urls:
-    - https://webhooks.gitter.im/e/7af45001fe6b7a4039f2
-    on_success: change
-    on_failure: always
-    on_start: never
```

Comment:
yup, it's just that the URL in this case is the same as for manageiq. 
can you give me one for the amazon room?
then I'll add it in this PR